src/internal/styles/typography/mixins.scss

```diff
@@ -94,7 +94,7 @@
 }
 
 @mixin font-button {
-  font-weight: awsui.$font-button-weight;
+  font-weight: awsui.$font-weight-button;
```

Comment:
I think we also need to consider what the expected behavior is for `<Button href="#" variant="link">`, and even more so `<Button href="#" variant="inline-link">`: should the font-weight of those be tied to that of normal buttons, or are they visually distinct enough entities that they should have their own design token(s)?

Reply:
Per offline discussion leaving this as is.